astropy/wcs/src/astropy_wcs.c

```diff
@@ -799,7 +780,8 @@ static PyTypeObject WcsType = {
 int _setup_wcs_type(
     PyObject* m) {
 
-  if (PyType_Ready(&WcsType) < 0)
+  WcsType = PyType_FromSpec(&WCSBaseClass_spec);
+  if (WcsType == NULL)
     return -1;
 
   Py_INCREF(&WcsType);
```

Comment:
This needs to be `Py_INCREF(WcsType);` now. And the line below it needs a similar change

Reply:
Actually, that's not true...

You no longer need the `INCREF` because `WcsType` is created with a refcount of 1.

The line below it does need to remove the `&`